packages/components/pagination/src/pagination.ts

```diff
@@ -234,6 +234,15 @@ export default defineComponent({
       emit('next-click', currentPageBridge.value)
     }
 
+    function addClass(element: any, cls: string) {
+      if (element) {
+        if (!element.props) {
+          element.props = {}
+        }
+        element.props.class = [element.props.class, cls].join(' ')
+      }
+    }
```

Comment:
I don't think changing the props here is a good idea, can we use `provide/inject` to do the same thing here?

Reply:
Can we do something else here? Exposing API to the parent for mutating the state.

```ts
element.someMethod()
```

Reply:
This method is modifies the class of the vnode, so I don't think it's necessary to exposing API